money-core/src/main/scala/com/comcast/money/emitters/LogEmitter.scala

```diff
@@ -38,34 +35,48 @@ object LogEmitter {
       case _ => Props(classOf[LogEmitter], conf)
     }.get
 
-  def buildMessage(t: Span): String = {
+  def buildMessage(t: Span)(implicit conf: Config): String = {
     implicit val builder = new StringBuilder()
-    builder.append("Span: ")
-    append("span-id", t.spanId.spanId)
-    append("trace-id", t.spanId.traceId)
-    append("parent-id", t.spanId.parentSpanId)
-    append("span-name", t.spanName)
-    append("app-name", Money.applicationName)
-    append("start-time", t.startTime)
-    append("span-duration", t.duration)
-    append("span-success", t.success)
+    builder.append(configValue("span-start", "Span: "))
```

Comment:
My concern is having to do all that checking every time we build a message.  Would be better if the config was parsed out to values and usable.

The config only needs to be inspected when the system starts up.

The above is one way to do it, I am sure there are other ways.